client/src/Movies/MovieList.js

```diff
@@ -24,7 +25,9 @@ export default class MovieList extends Component {
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+          <Link to={`/movies/${movie.id}`}>
+            <MovieDetails key={movie.id} movie={movie} />
```

Comment:
nice job, but you only need a key on the top level, which you have on MovieDetails